### PR TITLE
README: Explain updating ghu_toolkits submodule

### PR DESCRIPTION
**Pull request by ausbin**
Explain the approach I suggest in #2 in the README.

---

Explain how (and why you'd need to) update the ghu_toolkits submodule.
This is messy, but it's the best we can do with git, I think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,15 +6,28 @@ This is the Humanitarians United website prototype.
 Setup
 -----
 
+    $ git submodule update --init
+    $ git submodule update --remote
```
Comment (ausbin) [inline README.md:10]: You may be able to do these two in one line, let me check

Comment (ausbin) [reply README.md:10]: oh heck haha

Comment (S-reinken) [reply README.md:10]: No need. I like them separate for now.

Comment (ausbin) [reply README.md:10]: Sweet, thanks!

**Pull request merged**